src/programs/project3d/project3d.cpp

```diff
@@ -85,13 +87,13 @@ void Project3DApp::DoInteractiveUserInput( ) {
     delete my_input;
 
     //	my_current_job.Reset(14);
-    my_current_job.ManualSetArguments("tttiifffftbbbbbfi", input_star_filename.ToUTF8( ).data( ),
+    my_current_job.ManualSetArguments("tttiifffftbbbbbfit", input_star_filename.ToUTF8( ).data( ),
                                       input_reconstruction.ToUTF8( ).data( ),
                                       ouput_projection_stack.ToUTF8( ).data( ),
                                       first_particle, last_particle,
                                       pixel_size, mask_radius, wanted_SNR, padding,
                                       my_symmetry.ToUTF8( ).data( ),
-                                      apply_CTF, apply_shifts, apply_mask, add_noise, project_based_on_star, angular_step, max_threads);
+                                      apply_CTF, apply_shifts, apply_mask, add_noise, project_based_on_star, angular_step, max_threads, output_star_file.ToUTF8().data());
```

Comment:
I know you didn't change this here, but since you are modifying this line, it would be better to have either all inputs on a single line (generally for functions with only a few inputs) or in this case, to have only one parameter per line.

Having mixed approaches can make it much harder to spot bugs. Text alignment is a pretty useful tool in general.